### PR TITLE
flake8 and PEP fixes

### PR DESCRIPTION
**Pull request by rowanseymour**
Added flake8 with a config that ignores all but a few errors.. which have then been fixed in this PR. Moving forward, when someone gets the time, they can pop a few errors off the exclusion list, and fix them. This seems more realistic than trying to fix them all in one un-reviewable mega PR.

### CODE REVIEW

File: temba/contacts/fields.py [hunk 1]
```diff
@@ -49,7 +49,7 @@ def render(self, name, value, attrs=None):
     def get_json(self, value):
 
         if 'user' not in self.__dict__:  # pragma: no cover
-            raise ValidationError(_("Omnibox requires a user, make sure you set one using field.set_user(user) in your form.__init__"))
+            raise ValueError("Omnibox requires a user, make sure you set one using field.set_user(user) in your form.__init__")
```
Comment (rowanseymour) [inline temba/contacts/fields.py]: Since `_` wasn't imported this was actually a syntax error

**Comment by nicpottier** [comment]
But we already have you to tell us our PEP errors? What do we need flake for?

**Comment by rowanseymour** [comment]
flake8 is like me but more comprehensive... and maybe less funny

**Comment by nicpottier** [comment]
That doesn't sound like an upgrade at all! :)

But thanks for this, good to have someone else to keep us honest.

**Pull request merged**